rust/operator-binary/src/superset_controller.rs

```diff
@@ -368,3 +504,109 @@ fn build_server_rolegroup_statefulset(
 pub fn error_policy(_error: &Error, _ctx: Context<Ctx>) -> Action {
     Action::requeue(Duration::from_secs(5))
 }
+
+pub fn append_authentication_volumes_and_volume_mounts(
+    authentication_class: &AuthenticationClass,
+    volumes: &mut Vec<Volume>,
+    volume_mounts: &mut Vec<VolumeMount>,
+) {
+    let authentication_class_name = authentication_class.metadata.name.as_ref().unwrap();
+
+    match &authentication_class.spec.provider {
+        AuthenticationClassProvider::Ldap(ldap) => {
+            if let Some(bind_credentials) = &ldap.bind_credentials {
+                let volume_name = format!("{authentication_class_name}-bind-credentials");
+                let volume_mount_path = format!("/secrets/{volume_name}");
```

Comment:
Typically we put these mounts under /stackable.

Reply:
`/stackable/secrets/{volume_name}` would be fine?

Reply:
That's fine by me.